src/main/java/io/confluent/connect/jdbc/source/JdbcSourceConnectorConfig.java

```diff
@@ -240,6 +239,16 @@
   public static final long TIMESTAMP_DELAY_INTERVAL_MS_DEFAULT = 0;
   private static final String TIMESTAMP_DELAY_INTERVAL_MS_DISPLAY = "Delay Interval (ms)";
 
+  public static final String QUOTE_SQL_IDENTIFIERS_CONFIG = "quote.sql.identifiers";
+  public static final String QUOTE_SQL_IDENTIFIERS_DEFAULT = QuoteMethod.ALWAYS.name().toString();
+  public static final String QUOTE_SQL_IDENTIFIERS_DOC =
+      "When to quote table names, column names, and other identifiers in SQL statements. "
+      + "For backward compatibility, the default is 'always'.";
+  private static final String QUOTE_SQL_IDENTIFIERS_DISPLAY = "Quote Identifiers";
+
+  private static final EnumRecommender QUOTE_METHOD_RECOMMENDER =
+      EnumRecommender.in(QuoteMethod.values());
+
```

Comment:
Can we move to a common location to avoid the duplication between source and sink?  Maybe a superclass?  (I think it would also be useful to see which configs are common to both source and sink.)

Reply:
There is no common class, and it's actually kind of difficult to add one because while some of the constants are similar, the way they're used in ConfigDef are different (e.g., different groups). There's also no good package for a base class between the source and sink code.